doc/help/settings.asciidoc

```diff
@@ -3158,8 +3158,8 @@ Default:
 - +pass:[utm_content]+
 
 [[window.hide_wayland_decoration]]
```

Comment:
Given that this didn't change, are you editing this file by hand? Don't, see the beginning of the file :wink: